_sources/cardano-crypto-class/2.1.5.0/meta.toml

```
@@ -0,0 +1,3 @@
+timestamp = 2024-06-03T14:31:53Z
+github = { repo = "IntersectMBO/cardano-base", rev = "994d5b4" }
```

Let's use the full sha for the release: 
```suggestion
github = { repo = "IntersectMBO/cardano-base", rev = "994d5b42e28b98e4a4a53607182b93b6390e5487" }
```